pastastore/store.py

```diff
@@ -1201,43 +1205,101 @@ def solve_models(
             if True, errors emerging from the solve method are ignored,
             default is False which will raise an exception when a model
             cannot be optimized
-        store_result : bool, optional
-            if True save optimized models, default is True
         progressbar : bool, optional
-            show progressbar, default is True
-        **kwargs :
+            show progressbar, default is True.
+        parralel: bool, optional
+            if True, solve models in parallel using ProcessPoolExecutor
+        max_workers: int, optional
+            maximum number of workers to use in parallel solving, default is
+            None which will use the number of cores available on the machine
+        **kwargs : dictionary
             arguments are passed to the solve method.
+
+        Notes
+        -----
+        Users of operating systems other than Linux should be aware that
+        parallel solving is experimental and may not work as expected.
+        To use parallel solving on Windows, the following code should be used:
+
+        >>> from multiprocessing import freeze_support
```

Comment:
Use :: and then an indented block to include a code snippet. Better than the >>> which sort of implies plain Python?